boards/airfy-beacon/Makefile.include

```diff
@@ -7,6 +7,7 @@ DEBUG_ADAPTER ?= stlink
 STLINK_VERSION ?= 2
 
 PROGRAMMER = openocd
```

Comment:
```suggestion
PROGRAMMER ?= openocd
```
Unrelated but this bugs me hahaha

Reply:
Mind sneaking this one in?

Reply:
I dropped the line. The default programmer (also openocd) is already set by the nrf51 common logic.